### PR TITLE
ATA2-1767 Create eslint rule to validate presents of required tags

### PR DESCRIPTION
**Pull request by rostislav-simonik-plc**
# Jira
https://plentific.atlassian.net/browse/ATA2-1767

## OSSA 
* https://plentific.atlassian.net/browse/OOSSA-430

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,14 +11,14 @@
     "eslint-utils": "^3.0.0"
   },
   "devDependencies": {
-    "@babel/core": "^7.15.5",
-    "@babel/eslint-parser": "^7.15.4",
-    "@babel/preset-env": "^7.15.4",
-    "@typescript-eslint/parser": "^4.31.0",
-    "babel-plugin-module-resolver": "^4.1.0",
-    "eslint": "^7.19.0",
+    "@babel/core": "^7.23.9",
```
Comment (rostislav-simonik-plc) [inline package.json:14]: https://plentific.atlassian.net/browse/OOSSA-215

File: package.json [hunk 2]
```diff
@@ -11,14 +11,14 @@
...
+    "@babel/eslint-parser": "^7.23.10",
```
Comment (rostislav-simonik-plc) [inline package.json:15]: Existing https://plentific.atlassian.net/browse/OOSSA-429

File: package.json [hunk 3]
```diff
@@ -11,14 +11,14 @@
...
+    "@babel/preset-env": "^7.23.9",
```
Comment (rostislav-simonik-plc) [inline package.json:16]: Existing https://plentific.atlassian.net/browse/OOSSA-222

File: package.json [hunk 4]
```diff
@@ -11,14 +11,14 @@
...
+    "@typescript-eslint/parser": "7.0.2",
```
Comment (rostislav-simonik-plc) [inline package.json:17]: Existing https://plentific.atlassian.net/browse/OOSSA-429

File: package.json [hunk 5]
```diff
@@ -11,14 +11,14 @@
...
+    "babel-plugin-module-resolver": "^5.0.0",
```
Comment (rostislav-simonik-plc) [inline package.json:18]: Existing https://plentific.atlassian.net/browse/OOSSA-316

File: package.json [hunk 6]
```diff
@@ -11,14 +11,14 @@
...
+    "eslint": "^8.57.0",
     "eslint-plugin-eslint-comments": "^3.2.0",
     "eslint-plugin-plentific": "file:.",
-    "typescript": "^4.4.2"
+    "typescript": "^5.3.3"
```
Comment (rostislav-simonik-plc) [inline package.json:22]: https://plentific.atlassian.net/browse/OOSSA-306

File: package.json [hunk 7]
```diff
@@ -11,14 +11,14 @@
...
-    "@babel/preset-env": "^7.15.4",
-    "@typescript-eslint/parser": "^4.31.0",
-    "babel-plugin-module-resolver": "^4.1.0",
-    "eslint": "^7.19.0",
+    "@babel/core": "^7.23.9",
+    "@babel/eslint-parser": "^7.23.10",
+    "@babel/preset-env": "^7.23.9",
+    "@typescript-eslint/parser": "7.0.2",
+    "babel-plugin-module-resolver": "^5.0.0",
+    "eslint": "^8.57.0",
```
Comment (rostislav-simonik-plc) [inline package.json:19]: https://plentific.atlassian.net/browse/OOSSA-430

**Review by devendersharma-mset — APPROVED** [review]

**Review by devender-plentific — APPROVED** [review]

File: README.md [hunk 8]
```diff
@@ -40,3 +40,12 @@ To reverse this process, simply use `yarn unlink` or `yarn unlink eslint-plugin-
 
 Linting errors appears on fresh cloned project or code changes in linting rules files seem to not take an effect.
 * Remove `node_modules` and `run yarn`
+
+
+## Bumping the version 
+
+Once you finish your PR, don't forget bump the version in package.json, and also to reflect it in yarn.lock.
```
Comment (fbarbare) [inline README.md:47]: Changing this so it applies to all package managers

```suggestion
Once you finish your PR, don't forget bump the version in package.json, and also to reflect it in the lock file.
```

File: examples/ata/e2e/feature/foo/barFailure.test.ts [hunk 9]
```diff
@@ -0,0 +1,42 @@
+import { test, seed } from '../../../Test'
+
+seed(
+  `Seed title`,
+  {
+    tags: ['feature', 'foo', 'high'],
+
```
Comment (fbarbare) [inline examples/ata/e2e/feature/foo/barFailure.test.ts:7]:
```suggestion
```

File: examples/ata/e2e/feature/foo/barSuccess.test.ts [hunk 10]
```diff
@@ -0,0 +1,24 @@
+import { test, seed } from '../../../Test'
+
+seed(
+  'Seed title',
+  {
+    tags: ['feature', 'foo'],
+
```
Comment (fbarbare) [inline examples/ata/e2e/feature/foo/barSuccess.test.ts:7]:
```suggestion
```

**Review by fbarbare — APPROVED** [review]

**Pull request merged**